### ISSUE TITLE
Deactivate update_addons_current_version/fast_current_version crons

### ISSUE THREAD

**Issue by diox** [mozilla/addons#2281]
This is mostly lifted from https://bugzilla.mozilla.org/show_bug.cgi?id=956752 but re-explained with the latest investigation results and outside of bugzilla.

The `update_addons_current_version` / `fast_current_version` crons are nasty, going over all the database and spawning many tasks in an attempt to correct an old issue with the `_current_version` denormalized field on addons.

A long time ago the AMO team figured they should be useless by now and added a waffle switch to turn them off. Unfortunately, once that switch was flipped, regressions happened and it was never changed since - the switch is _on_ (at least in prod), which leaves the crons enabled.

From looking at Kibana it looks like these crons are useless nowadays, at least for non-personas. If I query for `"Version changed from current" AND (Hostname:"ip-172-31-47-71" OR Hostname:"ip-172-31-47-72" OR Hostname:"ip-172-31-37-161" OR Hostname:"ip-172-31-37-163")` in prod, I get no results, indicating that this log statement is never hit from celery ips. That statement is inside `Addon.update_version()`, which the cron task calls directly - that's all the task does, really. If you remove the hostname filtering you can see results, the log statement is hit regularly from our webservers when new versions are submitted/reviewed.

**TL;DR:** those crons can probably be deactivated.

Let's test that theory by turning the `current_version_crons` waffle switch _off_ in dev/stage and then prod soon.
